docs/src/operate/lifecycle/deregister.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Deregister entities
+tags: [Deregister]
```

Comment:
A better tag would be `Registration`. I also notice we have no tags related to chid-devices.

```suggestion
tags: [Child-Device, Registration]
```